spec/requests/api/alert_definitions_spec.rb

```diff
@@ -143,3 +144,115 @@
+      "description" => alert_definition_profile.description,
+      "guid"        => alert_definition_profile.guid
+    )
+  end
+
+  it "reads alert definitions subcollection of an alert definition profile" do
+    api_basic_authorize
+
+    alert_definitions = FactoryGirl.create_list(:miq_alert, 2)
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set, :alerts => alert_definitions)
+    run_get "#{alert_definition_profiles_url}/#{alert_definition_profile.id}/alert_definitions", :expand => "resources"
+    expect(response).to have_http_status(:ok)
+    expect_result_resources_to_include_hrefs(
+      "resources",
+      ["#{alert_definition_profiles_url}/#{alert_definition_profile.id}/alert_definitions/#{alert_definitions.first.id}",
+       "#{alert_definition_profiles_url}/#{alert_definition_profile.id}/alert_definitions/#{alert_definitions.first.id}"]
+    )
+  end
+
+  it "reads alert definition profile with expanded alert definitions subcollection" do
+    api_basic_authorize action_identifier(:alert_definition_profiles, :read, :resource_actions, :get)
+
+    alert_definition = FactoryGirl.create(:miq_alert)
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set, :alerts => [alert_definition])
+    run_get "#{alert_definition_profiles_url}/#{alert_definition_profile.id}", :expand => "alert_definitions"
+    expect(response).to have_http_status(:ok)
+    expect_single_resource_query(
+      "name" => alert_definition_profile.name, "description" => alert_definition_profile.description, "guid" => alert_definition_profile.guid
+    )
+    expect(response.parsed_body["alert_definitions"].first).to include(
+      "description" => alert_definition.description,
+      "guid"        => alert_definition.guid
+    )
+  end
+
+  it "creates an alert definition profile" do
+    sample_alert_definition_profile = {
+      "description" => "Test Alert Definition Profile",
+      "mode"        => "ContainerNode",
+    }
+    api_basic_authorize collection_action_identifier(:alert_definition_profiles, :create)
+    run_post(alert_definition_profiles_url, sample_alert_definition_profile)
+    expect(response).to have_http_status(:ok)
+    alert_definition_profile = MiqAlertSet.find(response.parsed_body["results"].first["id"])
+    expect(alert_definition_profile).to be_truthy
+    expect(response.parsed_body["results"].first).to include(
+      "description" => sample_alert_definition_profile["description"],
+      "mode"        => sample_alert_definition_profile["mode"]
+    )
+  end
+
+  it "deletes an alert definition profile via POST" do
+    api_basic_authorize action_identifier(:alert_definition_profiles, :delete, :resource_actions, :post)
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set)
+    run_post(alert_definition_profiles_url(alert_definition_profile.id), gen_request(:delete))
+    expect(response).to have_http_status(:ok)
```

Comment:
just esthetics/grouping, if you can add a blank line between the run_* commands and the expects, here and the tests below.  Thanks.